Author: n/a
Description: Disables to skip on encrypted disks (e.g. DVD with CSS-Encryption).

Index: dvdisaster/scsi-layer.c
===================================================================
--- dvdisaster.orig/scsi-layer.c	2012-03-06 11:10:13.371044799 +0900
+++ dvdisaster/scsi-layer.c	2012-03-06 11:10:17.147044691 +0900
@@ -2690,11 +2690,12 @@
        	 return NULL;
       }
    }
-
+/*
    if(dh->mainType == DVD && query_copyright(dh))
    {  CloseDevice(dh);
       Stop(_("This software does not support encrypted media.\n"));
    }
+*/
 
    /* Create the bitmap of simulated defects */
 
